shopfloor_reception_measuring_device/tests/test_reception_measuring_device.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo_test_helper import FakeModelLoader
+
+from odoo.addons.shopfloor_reception.tests.common import CommonCase
+
+class TestSetPackDimension(CommonCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.menu.sudo().set_packaging_dimension = True
+        cls.setUpClassPackaging()
+        cls.setUpComponentRegistry()
+        cls.setUpClassMeasuringDevice()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.loader.restore_registry()
+        return super().tearDownClass()
+
+    @classmethod
+    def setup_picking(cls):
+        return cls._create_picking(lines=[(cls.product_c, 10)])
+
+    @classmethod
+    def setUpComponentRegistry(cls):
+        from .device_component import MeasuringComponent
+        MeasuringComponent._build_component(cls._components_registry)
+
+    @classmethod
+    def setUpClassPackaging(cls):
+        cls.packaging1 = cls.product_c.packaging_ids
+        cls.packaging2 = cls.env["product.packaging"].sudo().create(
+            {
+                "name": "Big Box",
+                "product_id": cls.product_c.id,
+                "barcode": "ProductCBigBox",
+                "qty": 6,
+            }
+        )
+
+    @classmethod
+    def setUpClassMeasuringDevice(cls):
+        cls.loader = FakeModelLoader(cls.env, cls.__module__)
+        cls.loader.backup_registry()
+        from .device_model import MeasuringModel
+        cls.loader.update_registry((MeasuringModel, ))
+        cls.device_model = cls.env["measuring.device"].sudo()
+        cls.device = cls.device_model.create(
+            {
+                "name": "Test Device",
+                "device_type": "testdevice",
+                "state": "ready",
+                "warehouse_id": 1,
```

Comment:
```suggestion
                "warehouse_id": cls.warehouse.id
```
or whatever var we have